packages/web-components/src/components/va-memorable-date/va-memorable-date.scss

```diff
@@ -52,6 +52,25 @@
 @import '../va-date/va-date.css';
 @import '../../mixins/form-field-error.css';
 
+// Resets error border to not show for 'any' error, so we can only show for 'specific' errors
+:host([error]:not([error='']):not([uswds])) va-select::part(select),
```

Comment:
[A new issue was reported](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2076) that identified that `:not([uswds])` doesn't handle other variations of setting the prop so we should probably start doing `:not([uswds="true"])` instead to account for setting the prop as different booleans.